pynestml/codegeneration/printers/ode_toolbox_function_call_printer.py

```diff
@@ -89,7 +89,7 @@ def _print_function_call_format_string(self, function_call: ASTFunctionCall) ->
             return 'exp({!s})'
 
         if function_name == PredefinedFunctions.LN:
-            return 'log({!s})'
+            return 'ln({!s})'
 
         if function_name == PredefinedFunctions.LOG10:
             return 'log10({!s})'
```

Comment:
```suggestion
            return 'log({!s, 10})'
```
sympy.log10() does not exist. Do you know why this was working before?